SlicerCART/documentation/videotutorials.md

```diff
@@ -0,0 +1,10 @@
+# Video Tutorials
```

Comment:
Did you put the videos up to date ? Last time I checked they weren't and it can be misleading to have videos that are not up to date. I would also recommend to put the date at which the video was made so people can now how relevant it is.

Reply:
good point. yes. this is a basic video showing with the actual code as of 18 november. if you have any suggestion, please add. accepted (added the date)